<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateExportView</class>
 <widget class="QWidget" name="TemplateExportView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>25</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="UI_ViewResource.qrc">
     <normaloff>:/View/Icon/left_arrow.png</normaloff>:/View/Icon/left_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="thanks_text_label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>25</y>
     <width>780</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>Thanks for using our service</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="template_show_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>150</y>
     <width>1030</width>
     <height>1030</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 127);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="scan_text_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>1200</y>
     <width>1030</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Scan QR code to download</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="qr_code_image_label">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>1275</y>
     <width>1030</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="restart_button">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>1795</y>
     <width>1030</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>TAP TO RESTART</string>
   </property>
   <property name="icon">
    <iconset resource="UI_ViewResource.qrc">
     <normaloff>:/View/Icon/restart_icon.png</normaloff>:/View/Icon/restart_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="UI_ViewResource.qrc"/>
 </resources>
 <connections/>
</ui>
